src/families/stellar/js-getTransactionStatus.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import {
+  AmountRequired,
+  NotEnoughBalance,
+  FeeNotLoaded,
+  InvalidAddress,
+  InvalidAddressBecauseDestinationIsAlsoSource,
+  NotEnoughSpendableBalance,
+  NotEnoughBalanceBecauseDestinationNotCreated,
+  RecipientRequired,
+} from "@ledgerhq/errors";
+import {
+  StellarWrongMemoFormat,
+  SourceHasMultiSign,
+  AccountAwaitingSendPendingOperations,
+} from "../../errors";
+import { formatCurrencyUnit } from "../../currencies";
+import type { Account } from "../../types";
+import type { Transaction } from "./types";
+import {
+  isAddressValid,
+  checkRecipientExist,
+  isAccountMultiSign,
+  isMemoValid,
+} from "./logic";
+
+const getTransactionStatus = async (a: Account, t: Transaction) => {
+  const errors = {};
+  const warnings = {};
+  const useAllAmount = !!t.useAllAmount;
+
+  if (a.pendingOperations.length > 0) {
+    throw new AccountAwaitingSendPendingOperations();
+  }
+
+  if (!t.recipient) {
+    errors.recipient = new RecipientRequired("");
+  } else if (a.freshAddress === t.recipient) {
+    errors.recipient = new InvalidAddressBecauseDestinationIsAlsoSource();
+  } else if (!isAddressValid(t.recipient)) {
+    errors.recipient = new InvalidAddress("");
+  }
+
+  if (await isAccountMultiSign(a)) {
+    errors.recipient = new SourceHasMultiSign("", {
+      currencyName: a.currency.name,
+    });
+  }
+
+  if (!t.fees || !t.baseReserve) {
+    errors.fees = new FeeNotLoaded();
+  }
+
+  let estimatedFees = !t.fees ? BigNumber(0) : t.fees;
+  let baseReserve = !t.baseReserve ? BigNumber(0) : t.baseReserve;
+
+  let amount = !useAllAmount
+    ? t.amount
+    : a.balance.minus(baseReserve).minus(estimatedFees);
+  let totalSpent = !useAllAmount
+    ? amount.plus(estimatedFees)
+    : a.balance.minus(baseReserve);
+
+  if (totalSpent.gt(a.balance.minus(baseReserve))) {
+    errors.amount = new NotEnoughSpendableBalance(null, {
+      minimumAmount: formatCurrencyUnit(a.currency.units[0], baseReserve, {
```

Comment:
for this, it's ok, it's the only legit case to use formatter as it's meant for user display purpose. you should not disable the grouping.